app/helpers/brexit_checker_helper.rb

```diff
@@ -1,25 +1,94 @@
 require "addressable/uri"
 
 module BrexitCheckerHelper
-  def format_criteria_list(criteria)
-    criteria.map { |criterion| { readable_text: criterion.text } }
+  def extract_criteria(object)
```

Comment:
This helper does an awful lot.

Could we split it up in a few classes in ```lib/brexit_checker```? There could be one to do the grouping, one to handle the criterion extraction and then maybe a helper to present the data

Reply:
Can we catch up on this?